alpha/apps/kaltura/modules/kmcng/actions/kmcngAction.class.php

```diff
@@ -176,6 +176,7 @@ private function initConfig($deployUrl, $kmcngParams, $enforceSecureProtocol, $r
 		}
 
 		$config = array(
+			'ks' =>  $_SERVER['REQUEST_METHOD'] == 'POST' ? $_COOKIE['kmcks'] : null,
```

Comment:
Use builtin methods to get request method and cookies, 
$this->getRequestMethods()
$this->getRequest()->getCookie('kmcks')